src/test/resources/AuditConfigDb.properties

```diff
@@ -0,0 +1,4 @@
+dbName=Local MySQL Trial Direct
```

Comment:
This filename should be AuditConfigDb.properties.template
The AuditConfigDb.properties file is then created from it on each persons development environment.